content/en/docs/reference/glossary/webhook.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Webhook
+id: webhook
+date: 2019-02-13
+full_link: /docs/reference/access-authn-authz/webhook/
+short_description: >
+  A way for one component to notify another component in real time, via HTTP.
+
+aka:
+tags:
+- networking
+---
+ A notification message sent from one component to another component's unique URL,
+ via HTTP.
+
+<!--more-->
+
+The two components need not be related, and often are not. The sender need only
+know the URL of the recipient side. Typically, a webhook makes a `POST` request
+that includes a payload in JSON format. On the recipient side, the webhook can
+trigger further processing, such as invalidating a cache and refetching.
+
+In Kubernetes, “webhook mode” refers to calling an external service via HTTP.
```

Comment:
```suggestion
In Kubernetes, certain components such as Admission Controllers are able to interact with external to Kubernetes services via webhook.
```

Reply:
I'm keen to emphasis the difference between early definitions of the webhook concept and the way Kubernetes uses the term.
Is there a way to keep that distinction highlighted?

Reply:
Is there a "webhook mode" of Kubernetes? I figured that was just a plugin offered by the API Server for admission control.

Reply:
What I want to explain is that Kubernetes overall uses “webhook” to mean something slightly different that the original / more common meaning.

The admission controller in question is part of Kubernetes, right? It's not a core component though.